docs/DeveloperGuide.md

```diff
@@ -385,6 +388,145 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
       Use case resumes from step 2.
 
+**Use case: UC00 - Add home-buyer to user’s contact list**
+
+**MSS:**
+
+1. User chooses to add home-buyer.
+2. EstateEase requests for the details of the home-buyer.
+3. User enters the requested details.
+4. EstateEase adds the home-buyer and displays the newly added home-buyer. <br>
+   Use case ends.
+
+**Precondition for Extension 3i:** EstateEase has received the details of the home-buyer from the user. <br>
+**Trigger:** EstateEase validates the entered details and detects block number, street name, unit number and postal code in the entered data when he is a home-buyer.
+
+**Extensions**
+
+* 3a. EstateEase detects missing name in the entered data. <br>
+    * 3a1. EstateEase shows an error message regarding missing name. <br>
+      Use case resumes from step 2.
+  
+* 3b. EstateEase detects duplicate name in the entered data. <br>
+    * 3b1. EstateEase shows an error message regarding duplicate name. <br>
+      Use case ends.
+
+* 3c. EstateEase detects missing phone number in the entered data. <br>
+    * 3c1. EstateEase shows an error message regarding missing phone number. <br>
+      Use case resumes from step 2.
+
+* 3d. EstateEase detects incorrect format for phone number in the entered data. <br>
+    * 3d1. EstateEase shows an error message regarding incorrect format for phone number. <br>
+      Use case resumes from step 2.
+
+* 3e. EstateEase detects missing email in the entered data. <br>
+    * 3e1. EstateEase shows an error message regarding missing email. <br>
+      Use case resumes from step 2.
+
+* 3f. EstateEase detects incorrect format for email in the entered data. <br>
+    * 3f1. EstateEase shows an error message regarding incorrect format for email. <br>
+      Use case resumes from step 2.
+
+* 3g. EstateEase detects missing housing type (requirement for filter) in the entered data. <br>
+    * 3g1. EstateEase shows an error message regarding missing housing type. <br>
+      Use case resumes from step 2.
+
+* 3h. EstateEase detects incorrect housing type (requirement for filter) in the entered data. <br>
+    * 3h1. EstateEase shows an error message regarding the entry of incorrect of housing type. <br>
+      Use case resumes from step 2.
+
+* 3i. EstateEase detects block number, street name, unit number and postal code in the entered data. <br>
+    * 3i1. EstateEase shows an error message regarding the entry of housing details because home-buyer should not have a home yet. <br>
+      Use case resumes from step 2.
+
+* 3j. EstateEase detects missing role in the entered data. <br>
+    * 3j1. EstateEase shows an error message regarding missing role. <br>
+      Use case resumes from step 2.
+
+* 3k. EstateEase detects incorrect role in the entered data. <br>
+    * 3k1. EstateEase shows an error message regarding the entry of incorrect role. <br>
+      Use case resumes from step 2.
+
+**Use case: UC00 - Add more homes to home-seller**
```

Comment:
😂I think renumber all the UC instead. First one should be UC01 not UC00.

Reply:
Yeap updated the numbering of the use case. Thanks for reminding.